plasma_framework/test/endToEndTests/PaymentExitGame.e2e.test.js

```diff
@@ -283,7 +284,7 @@ contract('PaymentExitGame - End to End Tests', ([_deployer, _maintainer, authori
                 });
```

Comment:
Can we add a test cases that retrieves multiple `PaymentExitDataModel.StandardExit`s. Wonder if there is any upper bound to return arrays of structs. Can we test with 10,100,1000?